### DISCUSSION TITLE
Does Rows Grouping can be done with Polymorphic Relation column?

### DISCUSSION THREAD

**Discussion by kuchienkz** [filamentphp/filament#12089] [Help]
### Package

Table builder

### Package Version

v3.2.60

### How can we help you?

Based on the docs, it is possible to group rows by relation column with dot-syntax like `author.name`. However when I tried it with a Polymorphic relation, it gives "SQLSTATE[42S22]: Column not found: 1054" error. 

This is my models:

**Indication**
```
class Indication extends Model
{
    use HasFactory;

    protected $guarded = [];

    public function indicatable(): MorphTo
    {
        return $this->morphTo();
    }
}
```

**ModelA**
```
class ModelA extends Model
{
    use HasFactory;

    protected $guarded = [];

    public function indications(): MorphMany
    {
        return $this->morphMany(Indication::class, 'indicatable');
    }
}
```

**ModelB**
```
class ModelA extends Model
{
    use HasFactory;

    protected $guarded = [];

    public function indications(): MorphMany
    {
        return $this->morphMany(Indication::class, 'indicatable');
    }
}
```

This is what i did on the **IndicationResource**:
```
...
return $table
        ->defaultGroup(Group::make('indicatable.code')
        ->titlePrefixedWithLabel(false)
        ->collapsible())
...
```

This gives error:
**SQLSTATE[42S22]: Column not found: 1054 Unknown column 'indications.code' in 'field list'**
```
select * from `indications` order by (select `indications`.`code` from `indications` as `laravel_reserved_0` where `laravel_reserved_0`.`` = `indications`.`indicatable_id`) asc limit 10 offset 0
```

Did I miss something? Does Filament v3 support rows grouping with polymorphic relation?

**Comment by kablin** [comment]
Have same problem.

**Comment by Amiejah** [comment]
DId you ever find a solution for this?
